source/documentation/data-docs/curated-databases-docs/databases-for-analysis-and-apps.md

```diff
@@ -0,0 +1,56 @@
+# Using Databases and data for Apps
+
+This section covers how and when to use different tools to query our databases on the Analytical Platform and what to consider when giving data to a deployed app. This section assumes you are getting data from databases that are already created on the Analytical platform. If you need to upload data to the platform see the sections on [S3](#s3) and [information governance](#information-governance).
+
+## Guidance on using our databases for analysis
+
+We use `pydbtools` (for Python) and `dbtools` (for R) on the Analytical Platform to query our databases (You can find out more information about them in the [dbtools section](#dbtools)). Both these tools use an [SQL](#sql) Engine called [Athena](#amazon-athena) to query large datasets in S3 just like a normal relational database system. When you want to manipulate or query the databases on the Analytical Platform consider the following:
+
+- A good general rule is to do any aggregates, joins filtering in SQL on our databases (using Athena) first and then bring the resulting table into R/Python for more nuanced analytical calculations / transforms. If you just need to aggregate, join or filter your data and not do anything special with it then it might be worth considering doing all of your transforms using SQL Athena.
+
+- Is the data you are querying large? If so you might want to consider aggregating it or only selecting the columns you need using SQL (Athena) first before you read the data into an R/Python dataframe. R/Python are going to read that data into memory so that at some point of the dataset is too large is will kill your AP instance (on the flip side Athena takes that load for you so better to use Athena for big table queries and joins).
+
+- If you need to do joins across datasets it is most likely that Athena is going to be much faster than Python/R unless your datasets are very small. So if you are doing some analysis with many joins use Athena.
+
+If in doubt about best practices we have slack channels for R, python, SQL, etc where you can ask.
+
+## Guidance on using databases / data for deployed apps
+
+This section of the guidance covers best practice for using data on the platform for deployed apps. If you want to know how to set up an app bucket go to the [deploy an R shiny app section](#deploying-an-r-shiny-app) of the guidance.
+
+### What to consider when giving an app access to data
+
+This is because apps are typically accessed by users without Analytical Platform accounts. To add additional security for our data, we like to restrict how much data apps can access, in order to prevent unwanted data leaks in the unlikely event of an app being compromised. In order to describe how this affects things when we give apps access to databases, we need to briefly touch on 'roles' within the Analytical Platform.
```

Comment:
I don't like the start of this sentence (I realise I probably wrote this bit now). I suggest taking out "This is because" and just making the flat statement "Apps are typically accessed by users without Analytical Platform accounts." or maybe "Apps are typically accessed by users without Analytical Platform accounts, so we have to be more careful about security", and starting the next sentence "To this end we like to restrict" etc and so on.